### PR TITLE
Errors should be shown when it is not possible to login

### PR DESCRIPTION
**Pull request by wellitonSouza**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bug Fix

* **What is the current behavior?** (You can also link to an open issue here)
Did not show error notification when trying to log in

* **What is the new behavior (if this is a feature change)?**
Shows error notification if login or password is wrong

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
dojot/dojot#1771

### CODE REVIEW

**Review by luisfelipefurlan — APPROVED** [review]

File: src/js/views/login/translations/pt_br.login.i18n.json [hunk 1]
```diff
@@ -6,5 +6,6 @@
   "User is required": "Usuário é Obrigatório",
   "5 characters minimum": "Mínimo de 5 caracteres",
   "Type the password": "Digite a senha",
-  "Password is required": "Senha é obrigatório"
-}
+  "Password is required": "Senha é obrigatório",
+  "Error": "Login ou senha inválidos"
```
Comment (mprevide) [inline src/js/views/login/translations/pt_br.login.i18n.json:10]: If was not possible communication with back? What's error will be showing? 
If the error is "Login ou senha inválidos" maybe Its better change for something like "Erro ao ao tentar se logar".

Comment (wellitonSouza) [reply src/js/views/login/translations/pt_br.login.i18n.json]: I changed the login error message and added the network error

**Comment by mprevide** [comment]
You need add more tests

File: src/js/views/login/View.jsx [hunk 2]
```diff
@@ -62,10 +70,14 @@ const LoginForm = ({
   handleChange,
   handleBlur,
   handleSubmit,
+  hasError,
+  msgError,
 }) => {
   const classes = useStyles();
   const { t } = useTranslation(['login', 'common']);
 
+  console.log(msgError);
```
Comment (mprevide) [inline src/js/views/login/View.jsx:79]: Remove console log

File: src/js/views/login/View.jsx [hunk 3]
```diff
@@ -31,6 +34,9 @@ const LoginView = ({ location, history }) => {
     } catch (e) {
       // TODO: Handle the exception more appropriately
       console.error(e.message);
+      hasError = true;
+      msgError =
+        e.message.indexOf('404') !== -1 ? 'networkError' : 'loginError';
```
Comment (mprevide) [inline src/js/views/login/View.jsx:39]: Did you test this?

Comment (wellitonSouza) [reply src/js/views/login/View.jsx:39]: yes, i tested

**Review by mprevide — DISMISSED** [review]

**Review by luisfelipefurlan — APPROVED** [review]

**Review by luisfelipefurlan — APPROVED** [review]

**Review by mprevide — APPROVED** [review]

**Review by cfrancisco — CHANGES_REQUESTED** [review]
We should create a test to match the 2 possible login error: network error and login error.

**Comment by mprevide** [comment]
The tests **should be done** and create in a New PR!
I will merge this by request @bizetti.

**Pull request merged**